blogposts/2023-04-27-announcing-macos-71.md

```diff
@@ -0,0 +1,55 @@
+---
+post_title: "Introducing React Native macOS 0.71"
+author1: sanajmi
+post_slug: 2023-04-27-announcing-macos-71
+post_date: 2023-04-27 12:00:00
```

Comment:
this is TBC; waiting for Saad to complete his WP profile.

Reply:
Should be good now?

Reply:
yes